.github/pull_request_template.md

```diff
@@ -21,16 +21,19 @@ not fully covered by unit tests or manual testing can be complicated. -->
 <!-- Put an 'x' in all the boxes that apply -->
 - [ ] I have added unit tests to cover my changes.​
 - [ ] I have added integration tests to cover my changes.​
-- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
-- [ ] I have updated the [documentation](
-  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
+- [ ] I have added e2e tests for validation.
+- [ ] I have added the description of my changes into CHANGELOG in my target branch. (e.g., [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md) in develop)​
+- [ ] I have updated the documentation in my target branch accordingly. (e.g., [documentation](
+  https://github.com/openvinotoolkit/datumaro/tree/develop/site) in develop)
+- [ ] I have [linked related issues](
+  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
 
 ### License
 
 - [ ] I submit _my code changes_ under the same [MIT License](
   https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
   Feel free to contact the maintainers if that's a concern.
-- [ ] I have updated the license header for each file (see an example below)
+- [ ] I have updated the license header for each file. (see an example below)
```

Comment:
This must be `I have updated the license header for each file (see an example below).`